pkg/endpointmanager/conntrack.go

```diff
@@ -31,6 +31,7 @@ import (
 
 const (
 	// GcInterval is the garbage collection interval.
+	// #FIXME find a way to change this value
```

Comment:
The tests can become flake because they were supposed to count for the exact number of CT entries that were removed. With a garbage collector running every 10 seconds the number of CT entries that were removed need to be checked if they are less or equal than the expected number which IMO it's not very precise.

Reply:
This is now a variable that can be changed with `cilium config` it will be easier for the tests